mono/mini/mini-wasm-debugger.c

```diff
@@ -401,6 +401,9 @@ mono_wasm_setup_single_step (int kind)
 	if (!isBPOnNativeCode) {
 		mono_de_cancel_ss ();
 	}
+	if (depth == 0 && ss_req->global == TRUE) {
+		return 2;
```

Comment:
do we expect more return values here?  Having 0 and 2 need special handling but 1 be the normal case seems like an odd api choice. Maybe -1,0,1?

Reply:
I was planning to create a define, but it's okay -1, 0 and 1.